app/assets/stylesheets/arclight/modules/layout.scss

```diff
@@ -16,6 +16,21 @@
   font-size: $font-size-sm;
   height: 100%;
   padding: $spacer;
+
+  &-downloads {
+    padding: 0.32rem;
```

Comment:
The mocks seem to indicate download links aligned with the request button. Can we update some of the padding here to reflect that. And if its possible, can we use units based off `$spacing` or other defined variables where possible?

![Screen Shot 2019-09-18 at 7 43 02 AM](https://user-images.githubusercontent.com/1656824/65154105-251fec80-d9e8-11e9-96ab-fb99983c40f6.png)
![Screen Shot 2019-09-18 at 7 42 43 AM](https://user-images.githubusercontent.com/1656824/65154106-251fec80-d9e8-11e9-9cbf-e99c6a9011e4.png)